### DISCUSSION TITLE
Another weird boot loop problem

### DISCUSSION THREAD

**Discussion by Jakdaw** [alienatedsec/yi-hack-v5#320] [Q&A]
I seem to have a weird boot loop problem. Someone took a hammer to my old camera which has been running this firmware for months.

So I've another unit (bought at the same time, been in a cupboard for years!) onto which I got it onto wifi using the stock firmware then flashed 0.3.8 tried to update but seem to have gotten a boot loop. Thought there might be a problem with the SD card, so switched to a better one. I've used the reset button on the device so have to go through the QR code process with the app to tell it wifi details; whilst it reads the QR Code it never successfully connects to wifi, then resets and has lost it's configuration again (plays "Waiting to connect" message).

I've put the files from https://app.box.com/s/cibs7n1mgvhqaqjlidtveegu1uajt5yr/folder/63747474299 onto the SD card but that doesn't appear to have made any difference. Can anyone suggest next steps to try to figure out what's going wrong?

**Comment by alienatedsec** [comment]
It has to be FAT32 sd card. Please check of that is the case.

**Comment by Jakdaw** [comment]
Yes, I used mkfs.vfat
